app/models/deploy.rb

```diff
@@ -211,6 +211,10 @@ def url
     Rails.application.routes.url_helpers.project_deploy_url(project, self)
   end
 
+  def status_url
```

Comment:
can we not send this back or make it opt-in via `includes`, nobody else needs this